c10/util/complex_type.h

```diff
@@ -404,6 +404,38 @@ constexpr bool operator!=(const T& lhs, const c10::complex<T>& rhs) {
   return !(lhs == rhs);
 }
 
+
+// Define operators between integral scalars and c10::complex
```

Comment:
Nit: Could you have a line commenting that this behavior is different from `std::complex` but we feel it useful?

Reply:
Added